### PR TITLE
My spots

### PR DESCRIPTION
**Pull request by JanikOskar**

### CODE REVIEW

File: src/components/MySpots/MySpots.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+const CreateIcon = styled.img``;
+
+export const MySpots = () => {
+  return (
+    <>
+      <TopNav isWithBackArrow />
+      <HamburgerMenu isWithHamburger={false} />
+      <Wrapper>
+        <TextAndImg>
+          <img src={BoxInSpots} alt="dsa" />
```
Comment (lmazurowski) [inline src/components/MySpots/MySpots.tsx]: dsa ?

File: src/components/MySpots/MySpots.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  position: fixed;
+  bottom: 66px;
+  right: 18px;
+  background-color: ${props => props.theme.colors.main.primary};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const CreateIcon = styled.img``;
```
Comment (lmazurowski) [inline src/components/MySpots/MySpots.tsx:39]: why we are using empty wrapper here?

File: src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+const LinkRouterStyle = styled(NavLink)`
+  display: flex;
+  align-items: center;
+  color: ${props => props.theme.colors.main.primary}
+  height: 50px;
+  width:100%;
+  border-radius: 4px;
+  text-decoration: none
+  :hover
```
Comment (lmazurowski) [inline src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx]: I think this need to be format with indentation.

File: src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+const SearchInputStyled = styled.input`
+  width:100%
+  padding-left:32px;
+  background: #9db9ff;
+  height: 32px;
+  margin-right: 28px;
+  border: 1px solid rgba(255, 255, 255, 0.1);
+  box-sizing: border-box;
+  border-radius: 4px;
+  :: placeholder{
```
Comment (lmazurowski) [inline src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx]: I think this need to be format with indentation. Please use prettier/eslint.

File: src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+const LogoutButton = styled.div`
+  display: flex;
+  align-items: center;
+  color: ${props => props.theme.colors.main.primary}
+  height: 50px;
+  width:100%;
+  border-radius: 4px;
+  text-decoration: none
+  :hover
```
Comment (lmazurowski) [inline src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx]: same format issue here

File: src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+                  history.push('./login');
+                }}
+              >
+                <HamburgerIcon svgLink={LogoutSvg} />
+                <TextInHamburger>Logout</TextInHamburger>
+              </LogoutButton>
+            </DrawerWrapper>
+          </StyledDrawer>
+          {isWithSearch && (
+            <SearchPositionBox>
```
Comment (lmazurowski) [inline src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx]: We want to search something in the future? If yes - probably it is good to extract this component outside HamburgerMenu.

File: src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+              <SearchInputStyled placeholder="Search..." />
+              <SearchImg src={SearchIcon} alt="something " />
+            </SearchPositionBox>
+          )}
+          <RightProfile src={RightProfileSvg} alt="Something wrong" />
```
Comment (lmazurowski) [inline src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx]: Something wrong?

File: src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+              >
+                <HamburgerIcon svgLink={LogoutSvg} />
+                <TextInHamburger>Logout</TextInHamburger>
+              </LogoutButton>
+            </DrawerWrapper>
+          </StyledDrawer>
+          {isWithSearch && (
+            <SearchPositionBox>
+              <SearchInputStyled placeholder="Search..." />
+              <SearchImg src={SearchIcon} alt="something " />
```
Comment (lmazurowski) [inline src/global/Layouts/HamburgerMenu/HamburgerMenu.tsx]: something ?

File: src/global/graphqls/SignOut.ts [hunk 9]
```diff
@@ -0,0 +1,13 @@
+import { gql } from 'apollo-boost';
+
+export interface SignOutResponse {
```
Comment (lmazurowski) [inline src/global/graphqls/SignOut.ts]: this is not the correct BE response, when You log out. Are You sync with Your previous changes in git?

File: src/components/MySpots/MySpots.tsx (hunk 1 above)
Comment (JanikOskar) [reply src/components/MySpots/MySpots.tsx]: Change on Box In Spots

File: src/components/MySpots/MySpots.tsx (hunk 2 above)
Comment (JanikOskar) [reply src/components/MySpots/MySpots.tsx:39]: We dont need him.

**Review by laniszew — APPROVED** [review]

**Pull request merged**